apps/ejabberd/src/node_push.erl

```diff
@@ -207,20 +208,16 @@ is_allowed_to_publish(PublishModel, Affiliation) ->
 parse_form(undefined) ->
     #{};
 parse_form(Form) ->
-    IsForm = ?NS_XDATA == exml_query:attr(Form, <<"xmlns">>),
-    IsSubmit = <<"submit">> == exml_query:attr(Form, <<"type">>),
```

Comment:
@michalwski Fixed in #1234 :) @astro , could you please revert this change after #1234 is merged?